.vscode/extensions.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Please remove this file and `.vscode/settings.json`. While they're not bad per se, it's just not something we really add in Skyra Project repos.

Reply:
Yeah, I can do that. I mentioned it briefly to @kyranet though, so maybe you can find some nice solution for this overall.

> If I would have coded this with Codespaces, it would have used his .devcontainer settings and forced it on me this way
> I only added it since there is stuff like import sorting in there, which is kind of a source changing thing and will cause a conflict when you would re-save the file having those options enabled on your local settings

Reply:
You're not wrong. So far this hasn't been an issue though, so for now lets just revert it.

Reply:
Removed all unrelated changes 👍